### ISSUE TITLE
GLMAT_SOLVER instability with serial multi-mesh periodic bc

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#5269]
Marcos,
As we discussed, the attached case gives an instability in just a couple time steps using serial `intel_linux_64_dv` target.
[heated_channel_Pr_0p71_32.fds.txt](https://github.com/firemodels/fds/files/1165934/heated_channel_Pr_0p71_32.fds.txt)

**Comment by rmcdermo** [comment]
Marcos,
Please also add the GLMAT solver parameters to the PRES input list in the FDS User Guide.  You can point to a short writeup to Sec. 6.6 with some discussion about which solvers take precedence when various inputs are set.  Once you have this written up, I will use it to set up verification cases.
Thanks.

**Comment by marcosvanella** [comment]
Randy, I tried setting all Temperatures (WALL, initial and TMPA in &MISC) to 20.0C and taking out the volume heat source. The serial code run using PARDISO without problems for 1400 steps until I stopped it. 
This might have to do with the way divergence and mean pressure changes happen in a heated closed compartment, something I haven't dealt with before. We should be able to deal with a case of zero velocity field and heating or cooling + periodic bcs.
